.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+[*]
```

Comment:
Very minor, maybe add space as to seperate for better readability. This can be ignored as well.

```
root = true

[*]
```

Reply:
of course i'll change it. Much easier to just make the best readable.